novawallet/Modules/Staking/StartStakingInfo/RelaychainStaking/StartStakingInfoRelaychainPresenter.swift

```diff
@@ -0,0 +1,190 @@
+              let networkInfo = networkInfo.value,
+              let unstakePeriod = networkInfo?.stakingDuration.unlocking,
+              let nominationEraValue = nextEraTime(),
+              let minStake = minStake() else {
+            return
+        }
+        let maxApy = dashboardItem.maxApy
+        let title = startStakingViewModelFactory.earnupModel(
+            earnings: maxApy,
+            chainAsset: chainAsset,
+            locale: selectedLocale
+        )
+        let wikiUrl = startStakingViewModelFactory.wikiModel(
+            url: URL(string: "https://novawallet.io")!,
+            chain: chainAsset.chain,
+            locale: selectedLocale
+        )
+        let termsUrl = startStakingViewModelFactory.termsModel(
+            url: URL(string: "https://novawallet.io")!,
+            locale: selectedLocale
+        )
+        let testnetModel = chainAsset.chain.isTestnet ? startStakingViewModelFactory.testNetworkModel(
+            chain: chainAsset.chain,
+            locale: selectedLocale
+        ) : nil
+
+        let govModel = chainAsset.chain.hasGovernance ? startStakingViewModelFactory.govModel(
+            stakingType: stakingType,
+            chainAsset: chainAsset,
+            locale: selectedLocale
+        ) : nil
+
+        let paragraphs = [
+            testnetModel,
+            startStakingViewModelFactory.stakeModel(
+                minStake: minStake,
+                nextEra: nominationEraValue,
+                chainAsset: chainAsset,
+                locale: selectedLocale
+            ),
+            startStakingViewModelFactory.unstakeModel(unstakePeriod: unstakePeriod, locale: selectedLocale),
+            startStakingViewModelFactory.rewardModel(
+                stakingType: stakingType,
+                chainAsset: chainAsset,
+                eraDuration: eraDuration,
+                locale: selectedLocale
+            ),
+            govModel,
+            startStakingViewModelFactory.recommendationModel(locale: selectedLocale)
+        ].compactMap { $0 }
+
+        let model = StartStakingViewModel(
+            title: title,
+            paragraphs: paragraphs,
+            wikiUrl: wikiUrl,
+            termsUrl: termsUrl
+        )
+        view?.didReceive(viewModel: .loaded(value: model))
+    }
+
+    private func minStake() -> BigUInt? {
+        guard let networkInfo = networkInfo.value,
+              let minNominatorBond = minNominatorBond.value,
+              let bagListSize = bagListSize.value else {
+            return nil
+        }
+
+        return networkInfo?.calculateMinimumStake(
+            given: minNominatorBond,
+            votersCount: bagListSize
+        )
+    }
+
+    private func nextEraTime() -> TimeInterval? {
+        guard let eraCountdownResult = eraCountdown.value, let eraCountdown = eraCountdownResult else {
+            return nil
+        }
+
+        return eraCountdown.timeIntervalTillStart(targetEra: eraCountdown.activeEra + 1)
```

Comment:
It should be currentEra+1 not the activeEra. Because we can have a case when activeEra!=currentEra but validators already stared electing.